keyboards/shorty/k1/keyboard.json

```
@@ -0,0 +1,40 @@
+{
+    "manufacturer": "Hawtkeys",
+    "keyboard_name": "Shorty K1",
+    "maintainer": "hawtkeys",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP1", "pin_b": "GP2"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

NKRO probably doesn't serve much purpose on this keyboard.
```suggestion
        "mousekey": true
```